README.md

```diff
@@ -16,22 +16,22 @@ Report app size violations given a valid App Thinning Size Report.
 
```

Comment:
We should make the general plugin description broader now that we support both iOS and Android. We can write "A Danger plugin used to report iOS and Android app size violations" and leave it at that. 

Also let's move the hyperlink to https://developer.apple.com/documentation/xcode/reducing-your-app-s-size to 'App Thinning Size Report' in the usage description for `flag_violations`.